docs/source/howto/installation_guide.rst

```diff
@@ -51,7 +51,12 @@ which will install pyAFQ locally in editible mode and include extensions.
 .. note::
 
   It is also recommended that you utilize python virtual environment and
-  package mangagement tools and begin with a clean environment.
+  package mangagement tools (e.g. conda) and begin with a clean environment.
```

Comment:
```suggestion
  package mangagement tools (e.g., conda) and begin with a clean environment.
```

Something useful that I learned is that "e.g." and "i.e." are _always_ followed by a comma.